ansible/roles/tilemaker/templates/bicycle/process-openmaptiles.lua

```diff
@@ -395,7 +402,6 @@ function way_function(way)
 				way:MinZoom(minzoom)
 			end
 			SetNameAttributes(way)
-			way:Attribute("class",h)
```

Comment:
Could you explain why this is necessary?

Reply:
Because it was a duplication of [line 362](https://github.com/buskerudbyen/proposal-visualizer/blob/ac5ad7f37c12a25c7f8ed5890fb8b0de03b4f6f5/ansible/roles/tilemaker/templates/bicycle/process-openmaptiles.lua#L362)